app/views/shared/_supporting_information.html.erb

```diff
@@ -115,6 +112,38 @@
       <p class='govuk-body'>Consultation is not applicable for proposed permitted development.</p>
     </div>
   </div>
+  <div class="govuk-accordion__section">
+    <div class="govuk-accordion__section-header">
+      <h2 class="govuk-accordion__section-heading">
+        <button class="govuk-accordion__section-button" id="audit-log">
+          Audit log
+        </button>
+        <span class="govuk-accordion__icon" aria-hidden="true"></span>
+      </h2>
+    </div>
+    <div class="govuk-accordion__section-content" aria-labelledby="audit-log">
+      <% if @audit.present? %>
+        <p class="govuk-body">
+          <% if @audit.activity_type.match("/*_validation_request_cancelled") %>
+            <%= render "audits/types/validation_request_cancelled", locals: { item: @audit } %>
+          <% elsif @audit.activity_type.include?("request") || @audit.activity_type.include?("document_received_at_changed") %>
+            <%= render "audits/types/#{@audit.activity_type}", locals: { item: @audit } %>
+          <% else %>
+            <%= render "audits/types/generic_audit_entry", locals: { item: @audit } %>
+          <% end %>
+          <p class="govuk-body">
+            <%= define_audit_user(@audit) %>
+          </p>
+          <p class="govuk-body">
+            <%= @audit.created_at.to_date %> at <%= @audit.created_at.strftime("%H:%M") %> 
+          </p>
+        </p>
+      <% end %>
+      <p class='govuk-body'>
+        <%= link_to "View all", planning_application_audits_path(@planning_application), class: "govuk-link" %>
```

Comment:
This should be `View all activity` or `View all audits` or something descriptive rather than the generic `View all` (all what?)

Reply:
I know it's the design but this is the sort of thing we must challenge, it's good practice and a WCAG rule https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html

Reply:
can do, never gave it a second thought but its a good point to keep in mind thanks 👌